manage_breast_screening/core/jinja2/components/appointment-status/template.jinja

```diff
@@ -13,7 +13,7 @@
     {% set action_url = check_in_url or url('mammograms:check_in', kwargs={'pk': appointment.pk}) %}
     <form action="{{ action_url }}" method="post" novalidate>
       <p class="nhsuk-u-margin-top-2 nhsuk-u-margin-bottom-0">
-        <button class="app-button--link">Check in</button>
+        <button class="app-button--link">Check in {{ appointment.participant.full_name }}</button>
```

Comment:
```suggestion
        <button class="app-button--link">Check in<span class="nhsuk-u-visually-hidden"> {{ appointment.participant.full_name }}</span></button>
```
This should be hidden I think.

Reply:
I was assuming this would also be useful for anyone viewing at large magnifications, although I just checked and it's still legible at 200% magnification as the columns are close together. Are you happy for me to make this visually hidden @edwardhorsford @rivalee ?